mercury/test_models.py

```diff
@@ -1,25 +1,32 @@
 from django.test import TestCase
-from mercury.models import SimulatedData
+from mercury.models import (
+    TemperatureSensor,
+    # AccelerationSensor,
+    # WheelSpeedSensor,
+    # SuspensionSensor,
+    # FuelLevelSensor,
```

Comment:
These can be added when tests are written for them (which really should be a part of this change set, since this changeset is adding a lot of new code that isn't covered).